okapi/core/Okapi.php

```diff
@@ -1030,21 +1022,47 @@ public static function logtypeid2name($id)
         # These names are officially documented and may never change!
 
         # Primary.
-        if ($id == 1) return "Found it";
-        if ($id == 2) return "Didn't find it";
-        if ($id == 3) return "Comment";
-        if ($id == 7) return "Attended";
-        if ($id == 8) return "Will attend";
+        if ($id == 1) {
+            return "Found it";
+        }
+        if ($id == 2) {
+            return "Didn't find it";
+        }
+        if ($id == 3) {
+            return "Comment";
+        }
+        if ($id == 7) {
+            return "Attended";
+        }
+        if ($id == 8) {
+            return "Will attend";
+        }
 
         # Other.
-        if ($id == 4) return "Moved";
-        if ($id == 5) return "Needs maintenance";
-        if ($id == 6) return "Maintenance performed";
-        if ($id == 9) return "Archived";
-        if ($id == 10) return "Ready to search";
-        if ($id == 11) return "Temporarily unavailable";
-        if ($id == 12) return "OC Team comment";
-        if ($id == 13 || $id == 14) return "Locked";
+        if ($id == 4) {
+            return "Moved";
+        }
+        if ($id == 5) {
+            return "Needs maintenance";
+        }
+        if ($id == 6) {
+            return "Maintenance performed";
+        }
+        if ($id == 9) {
+            return "Archived";
+        }
+        if ($id == 10) {
+            return "Ready to search";
+        }
+        if ($id == 11) {
+            return "Temporarily unavailable";
+        }
+        if ($id == 12) {
+            return "OC Team comment";
+        }
+        if ($id == 13 || $id == 14) {
+            return "Locked";
+        }
```

Comment:
Huge code inflation here. The original code much more readable.

Reply:
My comment is not undestandible without the previous lines. Suggest to open a separate issue for discussion.